editor/editor_themes.cpp

```diff
@@ -512,39 +603,26 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_stylebox("separator", "VSeparator", make_line_stylebox(separator_color, border_width, 0, true));
 
 	// Debugger
-	Ref<StyleBoxFlat> style_panel_debugger = make_flat_stylebox(dark_color_2, 4, 4, 4, 4);
+	Ref<StyleBoxFlat> style_panel_debugger = style_default->duplicate();
 	theme->set_stylebox("DebuggerPanel", "EditorStyles", style_panel_debugger);
 
-	Ref<StyleBoxFlat> style_tab_fg_debugger = make_flat_stylebox(dark_color_2, 10, 5, 10, 5);
-	Ref<StyleBoxFlat> style_tab_bg_debugger = make_flat_stylebox(dark_color_2, 10, 5, 10, 5);
-	style_tab_bg_debugger->set_draw_center(false);
+	Ref<StyleBoxFlat> style_tab_fg_debugger = style_tab_selected->duplicate();
+	style_tab_fg_debugger->set_border_width_all(0);
 
 	theme->set_stylebox("DebuggerTabFG", "EditorStyles", style_tab_fg_debugger);
-	theme->set_stylebox("DebuggerTabBG", "EditorStyles", style_tab_bg_debugger);
+	theme->set_stylebox("DebuggerTabBG", "EditorStyles", style_tab_unselected);
 
 	// LineEdit
-	Ref<StyleBoxFlat> style_line_edit = make_flat_stylebox(dark_color_1, 6, 4, 6, 4);
-	style_line_edit->set_border_width_all(border_width);
-	style_line_edit = change_border_color(style_line_edit, contrast_color_1);
-	Ref<StyleBoxFlat> style_line_edit_disabled = change_border_color(style_line_edit, dark_color_1);
-	style_line_edit_disabled->set_bg_color(Color(0, 0, 0, .1));
-	Ref<StyleBoxFlat> style_line_edit_focus = change_border_color(style_line_edit, highlight_color);
-	theme->set_stylebox("normal", "LineEdit", style_line_edit);
-	theme->set_stylebox("focus", "LineEdit", style_line_edit_focus);
-	theme->set_stylebox("read_only", "LineEdit", style_line_edit_disabled);
+	theme->set_stylebox("normal", "LineEdit", style_widget);
+	theme->set_stylebox("focus", "LineEdit", style_widget_focus);
+	theme->set_stylebox("read_only", "LineEdit", style_widget_disabled);
```

Comment:
I was wondering about the widget approach! now I see! so much saved lines
What do you think about the name: `style_interaction_box` (I Couldn't imagine what you mean with widget. Only know that name form android phone home screens... but know I know. but for new ppl this name could be confusing?)